// fails_with="assertion failed"
import std.assert;

function main() ~> Nothing {
    std::assert(false);
}
